lib/compress/index.js

```diff
@@ -4992,6 +4993,10 @@ def_optimize(AST_Call, function(self, compressor) {
                 }).optimize(compressor);
             }
             break;
+          case "Symbol":
+            if (compressor.option("unsafe_symbols"))
+                return make_node(AST_SymbolRef, self, exp).optimize(compressor);
+                break;
```

Comment:
This is almost it!

So, besides checking the option, you need to have a look at `self.args` to make sure it has one item, and that the item is an AST_String. This matches the correct thing you're looking for, and also makes sure you're not removing any side-effects someone might've put in there (unlikely, sure, but you'd be surprised!)

In this case, you don't even need make_node, since you're transforming a call expression into a call expression with zero arguments.

So, `self.args.length = 0` will do :)

Reply:
Alright, thanks! I've made the changes. I'll write up some tests for it shortly.